res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores/__init__.py

```diff
@@ -0,0 +1,8 @@
+import tensorflow as tf
+
+max_output_size = tf.compat.v1.constant(4)
+
+in_boxes_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(8, 4), name="Hole")
+in_scores_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(8), name="Hole")
+non_max_suppression_with_scores_ = tf.compat.v1.image.non_max_suppression_with_scores(
```

Comment:
Error with TF1.13.x
> AttributeError: module 'tensorflow._api.v1.compat.v1.image' has no attribute 'non_max_suppression_with_scores'

If we should run in higher versions, please add a note about it.

Reply:
It seems ``non_max_suppression_with_scores`` was introduced in TF 1.15, I've added comment about version requirement.